coordinator/tests/services/translation/common.spec.ts

```diff
@@ -1,107 +1,107 @@
 import {
-  getIdentifierValueForSystem,
-  getIdentifierValueOrNullForSystem,
-  getResourceForFullUrl,
-  getResourcesOfType,
-  wrapInOperationOutcome
+    getIdentifierValueForSystem,
+    getIdentifierValueOrNullForSystem,
+    getResourceForFullUrl,
+    getResourcesOfType,
+    wrapInOperationOutcome
 } from "../../../src/services/translation/common"
 import * as TestResources from "../../resources/test-resources"
 import * as fhir from "../../../src/model/fhir-resources"
 import {Identifier, MedicationRequest} from "../../../src/model/fhir-resources"
 import {clone} from "../../resources/test-helpers"
 import {SpineDirectResponse} from "../../../src/services/spine-communication"
 
-test('getResourcesOfType returns correct resources', () => {
-  const result = getResourcesOfType(TestResources.examplePrescription1.fhirMessageUnsigned, new MedicationRequest())
-  expect(result).toBeInstanceOf(Array)
-  expect(result).toHaveLength(4)
-  result.map(x => expect((x as fhir.Resource).resourceType).toBe("MedicationRequest"))
+test("getResourcesOfType returns correct resources", () => {
+    const result = getResourcesOfType(TestResources.examplePrescription1.fhirMessageUnsigned, new MedicationRequest())
+    expect(result).toBeInstanceOf(Array)
+    expect(result).toHaveLength(4)
+    result.map(x => expect((x as fhir.Resource).resourceType).toBe("MedicationRequest"))
 })
 
-test('getResourceForFullUrl returns correct resources', () => {
-  const result = getResourceForFullUrl(TestResources.examplePrescription1.fhirMessageUnsigned, "urn:uuid:A7B86F8D-1D81-FC28-E050-D20AE3A215F0")
-  expect((result as fhir.Resource).resourceType).toBe("MedicationRequest")
+test("getResourceForFullUrl returns correct resources", () => {
+    const result = getResourceForFullUrl(TestResources.examplePrescription1.fhirMessageUnsigned, "urn:uuid:A7B86F8D-1D81-FC28-E050-D20AE3A215F0")
+    expect((result as fhir.Resource).resourceType).toBe("MedicationRequest")
 })
 
-test('getResourceForFullUrl throws error when finding multiple resources', () => {
-  const bundle2 = clone(TestResources.examplePrescription1.fhirMessageUnsigned)
-  bundle2.entry[1].fullUrl = bundle2.entry[0].fullUrl
-  expect(() => getResourceForFullUrl(bundle2, bundle2.entry[0].fullUrl)).toThrow(TypeError)
+test("getResourceForFullUrl throws error when finding multiple resources", () => {
+    const bundle2 = clone(TestResources.examplePrescription1.fhirMessageUnsigned)
+    bundle2.entry[1].fullUrl = bundle2.entry[0].fullUrl
+    expect(() => getResourceForFullUrl(bundle2, bundle2.entry[0].fullUrl)).toThrow(TypeError)
 })
 
-describe('getIdentifierValueForSystem', () => {
-  const identifierArray: Array<Identifier> = [
```

Comment:
i think this is an example of a place which looks better with 2 spaces for indentation over 4 - maybe we try first with 2 and see if people dislike it?

Reply:
Happy to switch to two spaces. I presume you mean globally, rather than for this specific case?

Reply:
yeah if its not too much hassle to sort

Reply:
Done. Pushed up a new commit with everything set to two spaces